apps/vscode-extension/src/intellisense/design-token.ts

```diff
@@ -5,8 +5,40 @@ import * as path from 'node:path';
 
 const readFile = async (filePat: string) => (await vscode.workspace.fs.readFile(vscode.Uri.parse(filePat))).toString();
 
+const getFilesPatternsFromProjectConfiguration = async (): Promise<string[] | undefined> => {
+  const angularFile = await vscode.workspace.findFiles('angular.json');
+  if (angularFile.length === 0) {
+    return;
+  }
+
+  const walkThroughAngularFile = (node: any, mem: Set<string> = new Set()) => {
+    if (typeof node === 'object' && node !== null && !Array.isArray(node)) {
+      if (node.executor === '@o3r/design:generate-css' || node.builder === '@o3r/design:generate-css') {
+        node.options?.designTokenFilePatterns?.forEach((pattern: string) => mem.add(pattern));
+        if (node.options?.configurations) {
```

Comment:
which configurations are you targeting? Isn't the configurations node at the same level as node.options?
![image](https://github.com/AmadeusITGroup/otter/assets/110408057/befd9118-0914-4b80-a51d-6a3affa957ff)

Reply:
oops, I trusted too much copilot on this one :S